File: 2310817120009_AlysaArmelia_PemrogramanI_Modul1/Python/PRAK110-2310817120009-AlysaArmelia.py
import math
c = 12
a = 5
b = round (math.sqrt(a*a + c*c))

K = a + b + c
l = round (1 / 2 * a * c)

print("Diketahui : ")
print(f"Alas = {c} cm")
print(f"Tinggi = {a} cm ")
print("Jawab : ")
print(f"Sisi A = {a} cm")
print(f"Sisi B = {b} cm")
print(f"Sisi C = {c} cm")
print(f"Keliling = {K} cm")
print(f"Luas {l} cm")

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul1/Bahasa C/PRAK110-2310817120009-AlysaArmelia.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a = 5;
    int t = 12;
    int x = (a * a) + (t * t);
    int hasil = sqrt (x);
    printf("Diketahui ;\n");
    printf("Alas = %d cm\n", a);
    printf("Tinggi = %d cm\n", t);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n",t );
    printf("Sisi B = %d cm\n",a );
    printf("Sisi C = %d cm\n",hasil);
    printf("Keliling = %d cm\n",a+t+hasil);
    printf("Luas =%d cm\n", a*t/2);
return 0;
}